src/neki.jl

```diff
@@ -1,28 +1,28 @@
 """
-    neki_iter(prob::SolusProblem, θs, fθs)
+    neki_iter(prob::CESProblem, θs, fθs)
 
-Peform an iteration of NEKI, returning a new set of proposed `θs`.
+Perform an iteration of NEKI, returning a new set of proposed `θs`.
 
 See eqs. (5.4a-b) from https://arxiv.org/abs/1903.08866.
```

Comment:
Update reference, @agarbuno?

Reply:
Should we change this to `https://arxiv.org/pdf/2001.03689.pdf`?